src/main/webapp/js/util.js

```diff
@@ -80,3 +80,29 @@ function createDealCard(deal) {
   dealCard.appendChild(dealBody);
   return dealCard;
 }
+
+/**
```

Comment:
Should you also replace the copy of this you have in search or elsewhere in the codebase with this?

Reply:
Feel free to do this in a follow up PR.